apps/mobile/src/components/ProfileView/ProfileView.tsx

```diff
@@ -222,13 +226,21 @@ function ConnectedProfilePicture({ queryRef }: ConnectedProfilePictureProps) {
   );
 
   const bottomSheetRef = useRef<GalleryBottomSheetModalType | null>(null);
+  const { openManageWallet } = useManageWalletActions();
+  const userHasWallet = query.viewer?.user?.primaryWallet?.__typename === 'Wallet' ?? false;
```

Comment:
I don't think we need a ternary here because === will return true or false

Reply:
Removed!